src/pages/RepoPage/CoverageTab/subroute/FileExplorer/CodeTreeTable/CodeTreeTable.spec.tsx

```diff
@@ -123,12 +138,14 @@ const mockOverview = {
 }
 
 const wrapper =
-  (initialEntries = '/gh/codecov/cool-repo/tree/main/a/b/c') =>
+  (
+    initialEntries = '/gh/codecov/cool-repo/tree/main/'
+  ): React.FC<React.PropsWithChildren> =>
   ({ children }) => {
     return (
       <QueryClientProvider client={queryClient}>
         <MemoryRouter initialEntries={[initialEntries]}>
-          <Route path="/:provider/:owner/:repo/tree/:branch/:path+">
+          <Route path="/:provider/:owner/:repo/tree/:branch/:path*">
```

Comment:
We probably want to leave `+` here to align with the paths defined in parent components. As well, `react-router` uses regex to match these paths, and we want the `+` here so that it requires at minimum "one" or more paths, whereas `*` will match 0 or more.

Reply:
I think that is fine. The path may or may not be defined on the app as well. 
Example: https://app.codecov.io/gh/codecov/gazebo